tests/cypress/integration/dashboard/dashboard.cy.js

```diff
@@ -0,0 +1,137 @@
+/// <reference types="cypress" />
+
+import * as GeneralSetting from '../../pageAction/generalSetting.action';
+import * as AddProducts from '../../pageAction/addNewProduct.action';
+import * as CustomizeYourStore from '../../pageAction/customizeStore.action';
+import * as LaunchYourStore from '../../pageAction/launchYourStore.action';
```

Comment:
I'm not a fan of the page object/action approach. It obscures exactly what is being tested and adds a layer of complexity that impact maintainability and test performance.